src/styles/base/_reset.scss

```diff
@@ -0,0 +1,21 @@
+html,
+body,
+div,
+h1,
+h2,
+h3,
+p,
+span,
+ul,
+li,
+button {
+  margin: 0;
+  padding: 0;
+  list-style: none;
```

Comment:
The 'list-style: none;' property should only be applied to list elements like 'ul' and 'li'. Applying it to other elements like 'html', 'body', 'div', etc., is unnecessary and could be misleading.